blockchain/chain_sync/src/bad_block_cache.rs

```diff
@@ -17,7 +17,7 @@ pub struct BadBlockCache {
 
 impl Default for BadBlockCache {
     fn default() -> Self {
-        Self::new(forest_utils::const_option!(NonZeroUsize::new(1 << 15)))
+        Self::new(NonZeroUsize::new(1 << 15).unwrap())
```

Comment:
Can we use `nonzero!` here?

Reply:
It wouldn't do anything - see https://chainsafeio.slack.com/archives/C0144LHN732/p1684843121482679

We _could_ make this const, but it's just as readable (and probably just as performant) as is

Reply:
It would fail at compile time rather than runtime. This compiles just fine `NonZeroUsize::new(0 << 15).unwrap()`, whereas this does not `nonzero!(0_usize << 15)`.